crates/forge/src/cheatcodes_hint_processor/file_operations.rs

```diff
@@ -25,6 +27,43 @@ pub(super) fn parse_txt(file_path: &Felt252) -> Result<Vec<Felt252>, EnhancedHin
         })
 }
 
+pub(super) fn parse_json(file_path: &Felt252) -> Result<Vec<Felt252>, EnhancedHintError> {
+    let file_path_str = as_cairo_short_string(file_path)
+        .with_context(|| format!("Failed to convert {file_path} to str"))?;
+    let content = std::fs::read_to_string(&file_path_str)?;
+    let split_content = json_values_sorted_by_keys(&content).map_err(|_| FileParsing {
+        path: file_path_str.clone(),
+    })?;
+
+    let felts_in_results: Vec<Result<Felt252, ()>> = split_content
+        .iter()
+        .map(|string| string_into_felt(string))
+        .collect();
+
+    felts_in_results
+        .iter()
+        .cloned()
+        .collect::<Result<Vec<Felt252>, ()>>()
+        .map_err(|_| FileParsing {
+            path: file_path_str,
+        })
+}
+
+fn json_values_sorted_by_keys(content: &str) -> Result<Vec<String>, EnhancedHintError> {
+    let json: Map<String, Value> =
+        serde_json::from_str(content).map_err(|_| anyhow!("Wrong format"))?;
+
+    let data = flatten(&json);
+
+    let mut keys: Vec<String> = data.keys().map(std::string::ToString::to_string).collect();
+    keys.sort_by_key(|a| a.to_lowercase());
```

Comment:
I assume those keys are in the flatten format, namely `a.b.c`. I don't think it is explained anywhere in the docs how alphanumeric ordering works for nested objects

Reply:
I added note in docs